lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -89,12 +84,12 @@ def example_with_aggregate_failures?(example_node)
           node_with_aggregate_failures = find_aggregate_failures(example_node)
           return false unless node_with_aggregate_failures
 
-          aggregate_failures?(node_with_aggregate_failures)
+          aggregate_failures?(node_with_aggregate_failures, :true_type?.to_proc)
```

Comment:
the direct `to_proc` usage is very uncommon. I still wonder whether I prefer an explicit lambda

Reply:
Agree, changed to an explicit lambda just like `ANYTHING`.